src/draive/conversation/types.py

```
@@ -1,49 +1,127 @@
+    created: datetime = Default(factory=datetime.now)
+    content: MultimodalContent
+    eod: bool = False
+    meta: Meta = META_EMPTY
+
+    def __bool__(self) -> bool:
+        return bool(self.content)
+
+
 class ConversationMessage(DataModel):
+    @classmethod
+    def from_chunks(
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)